src/site/site.xml

```diff
@@ -5,6 +5,11 @@
     xsi:schemaLocation="http://maven.apache.org/DECORATION/1.8.0
                         https://maven.apache.org/xsd/decoration-1.8.0.xsd">
 
+  <skin>
+    <groupId>org.apache.maven.skins</groupId>
+    <artifactId>maven-fluido-skin</artifactId>
+    <version>2.0.0-M11</version>
```

Comment:
Use the GA version.

Reply:
@michael-o , I tried to use GA, but still the same result: no site pages are generated, only main index.html with empty content. see all details bellow.
in debug logs, there is nothing obviously suspicious.
 
@michael-o , do know what can cause this ?